### DISCUSSION TITLE
Date Format Issue with `issued` Field in References Extracted by Pandoc from Word (Zotero Plugin)

### DISCUSSION THREAD

**Discussion by jpmmartins** [jgm/pandoc#10360] [General]
When extracting references from a Word document created using the Zotero Plugin, Pandoc outputs the `issued` field as a plain string rather than in an array `date-parts`. The array representation is used in the the docx and it is also possible in the csl specification.  This causes compatibility issues with an application that we are using, which expects a structured date array. Is there a way of configuring the schema of the pandoc output?

**Steps to Reproduce**
1. Create a Word document with citations using the Zotero Plugin.
2. Extract the references to JSON format using Pandoc.
3. Observe the format of the `issued` field in the resulting JSON.

**Environment Details**
    •   Pandoc version: 3.5
    •   Zotero version: 6.0.36
    •   Operating system: WSL2 running Windows 10 22H2

**Comment by jgm** [comment]
Please upload a Word document I can test with, and let me know the exact pandoc command you used.

**Comment by jpmmartins** [comment]
I used a test document with 2 citations in the chicago-note citation style. The instruction that I used was the following

```
pandoc test_article.docx -f docx+citations --citeproc -t json -o test.json  --csl=chicago-note-bibliography.csl --standalone
```

The docx is here attached. [test_article.docx](https://github.com/user-attachments/files/17663909/test_article.docx)

The csl can be downloaded from [here](https://github.com/citation-style-language/styles/blob/master/chicago-note-bibliography.csl)

Many thanks for replying!

**Comment by jgm** [comment]
Here's what I am seeing.  If you convert to `csljson`, `"issued"` is properly structured.
```
% pandoc ~/Downloads/test_article.docx -t csljson -s -f docx+citations
[
  {
    "ISSN": "1070-986X",
    "author": [
      {
        "family": "Kogler",
        "given": "Rita Ann"
      }
    ],
    "container-title": "IEEE MultiMedia",
    "id": "309",
    "issue": "1",
    "issued": {
      "date-parts": [
        [
          1998
        ]
      ]
    },
    "page": "92–96",
    "title": "Virtual Set Design",
    "type": "article-journal",
    "volume": "5"
  },
  {
    "author": [
      {
        "family": "Levoy",
        "given": "Marc S."
      }
    ],
    "genre": "Master's Thesis",
    "id": "317",
    "issued": {
      "date-parts": [
        [
          1979,
          1,
          1
        ]
      ]
    },
    "publisher": "Cornell University",
    "title": "Computer-assisted cartoon animation",
    "type": "thesis"
  }
]
```
But if you convert to `native` (or `json`), you'll see:
```
                       , ( "issue" , MetaString "1" )

```
This is all as intended.  Pandoc represents dates as a single string in its metadata, and it's smart enough to create the structured date-parts on the fly when it passes the bibliography to citeproc.  I don't think this is going to cause any problems for you.

**Reply by jpmmartins** [reply]
I will see if we need to adapt our application to the output format or if we need to extract the references separately.

Many thanks for your reply, I really appreciate it.
